internal/tokenmanager/tokenmanager.go

```diff
@@ -65,6 +66,14 @@ func (m *singleTokenAccessTokenManager) DefaultToken() (string, error) {
 }
 
 // Token always returns the token managed by the singleTokenAccessTokenManager.
-func (m *singleTokenAccessTokenManager) Token(*commonpb.Identity) (string, error) {
+func (m *singleTokenAccessTokenManager) Token(identity interface{}) (string, error) {
```

Comment:
Some notes:
- can't use generics here: https://go.googlesource.com/proposal/+/refs/heads/master/design/43651-type-parameters.md#no-parameterized-methods
- can't use overloading: https://go.dev/doc/faq#overloading

Other option: have two methods: ```Token(*commonpbv1.Identity)``` and ```TokenV2(*commonpb.Identity)```